lib/acts_as_taggable_on/taggable/core.rb

```diff
@@ -315,3 +316,4 @@ def find_or_create_tags_from_list_with_context(tag_list, _context)
     end
   end
 end
+
```

Comment:
👍

Reply:
This one was fine actually, It's good to have a blank line at the end of files :). Sorry, I should have been more specific than a 👍  
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file